student.rb

```diff
@@ -1,15 +1,40 @@
+require_relative 'classroom'
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :name, :age, :parent_permission, :classroom
+
   def initialize(classroom, name = 'unknown', age = nil, parent_permission: true)
     super(name, age, parent_permission: parent_permission)
     @classroom = classroom
+    @name = name
+    @age = age
+    classroom.add_student(self)
   end
 
```

Comment:
![image](https://github.com/AbuTalha3/awesomeBooksList/assets/93039359/a9c01198-0764-4754-8d3c-579018b7b239)

- [ ] It would be nice if you fix the link highlighted in the image above to make your readme descriptive and professional 👍If you don't have any section for this link then kindly remove this link 😊